docker/startup.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+cd /xen-orchestra
+yarn
+yarn build
+yarn test-integration
```

Comment:
Maybe we should use `script/travis-tests`.

Reply:
I'm not very confident in "find related test", it misses quite a lot changes in dependencies

Reply:
I propose that we use it anyway for now and we'll improve the logic in another PR.

Also, I propose we create a `ci` script in `package.json` which does `yarn && script/travis-test`, this will help with future maintenance. 

What do you think?

Reply:
I really think test should be run completly , at least before any merge.

Reply:
This is another concern that we'll take care in the `ci` script, nothing to do with this setup.